pkgs/development/python-modules/nutils-poly/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  rustPlatform,
+  numpy,
+  unittestCheckHook,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "nutils-poly";
+  version = "1.0.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "nutils";
+    repo = "poly-py";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-dxFv4Az3uz6Du5dk5KZJ+unVbt3aZjxXliAQZhmBWDM=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    name = "${pname}-${version}";
+    inherit src;
+    hash = "sha256-+fnKvlSwM197rsyusFH7rs1W6livxel45UGbi1sB05k=";
+  };
+
+  nativeBuildInputs = [ rustPlatform.cargoSetupHook ];
+
+  build-system = [ rustPlatform.maturinBuildHook ];
+
+  dependencies = [ numpy ];
+
+  nativeCheckInputs = [ unittestCheckHook ];
+
+  pythonImportsCheck = [ "nutils_poly" ];
+
+  meta = {
+    description = "Low-level functions for evaluating and manipulating polynomials";
+    homepage = "https://github.com/nutils/poly-py";
+    license = lib.licenses.mit;
```

Comment:
It looks like this package doesn't have a license.
Could you ask upstream to set the license?

Reply:
Both
https://github.com/nutils/poly-py/blob/main/Cargo.toml
and
https://pypi.org/project/nutils-poly/
list the package as MIT.

Should I still ask that they add a `LICENSE` file to the repo?

Reply:
Thanks for letting me know. LGTM.